src/content.js

```diff
@@ -5,404 +5,414 @@ import { isNumber } from './js/utils/misc/isNumber';
+    initDarkMode,
+    refreshMutedPlayers,
+    displayInformationPopup
 } from './js/ui/content/functions';
 import { gamesWithTwoTeams } from "./js/config/darkThemeGames";
+import { initGamePanelObserver } from "./js/ui/content/gamePanel/ongoingGamesCount.ts";
 
 const config = new Configuration();
 let currentObserver = null;
 let pageType = undefined;
 let chatbardock = undefined;
 
 if (localStorage.getItem('ext_dark_theme') === 'on') {
-	// hack to avoid light theme flashing
-	const s = document.createElement('style');
-	s.innerHTML = "* { visibility: hidden; }"
-	document.documentElement.appendChild(s);
-	window.addEventListener("load", () => {
-		if (window.navigator.userAgent.toLowerCase().includes('firefox')) {
-			setTimeout(() => { s.remove(); }, 500);
-		} else {
-			s.remove();
-		}
-	});
+    // hack to avoid light theme flashing
+    const s = document.createElement('style');
+    s.innerHTML = "* { visibility: hidden; }"
+    document.documentElement.appendChild(s);
+    window.addEventListener("load", () => {
+        if (window.navigator.userAgent.toLowerCase().includes('firefox')) {
+            setTimeout(() => { s.remove(); }, 500);
+        } else {
+            s.remove();
+        }
+    });
 }
 
 const autoHideChat = (e) => {
-	if (chatbardock) {
-		const clientRect = chatbardock.getBoundingClientRect();
-		const minX = clientRect.x - 100;
-		const maxX = clientRect.x + clientRect.width + 100;
-		const minY = clientRect.y - 50;
-
-		if (e.clientX > minX && e.clientX < maxX && e.clientY > minY) {
-			chatbardock.classList.remove('bgaext_hidden');
-		} else {
-			chatbardock.classList.add('bgaext_hidden');
-		}
-	} else {
-		chatbardock = document.querySelector('#chatbardock');
-	}
+    if (chatbardock) {
+        const clientRect = chatbardock.getBoundingClientRect();
+        const minX = clientRect.x - 100;
+        const maxX = clientRect.x + clientRect.width + 100;
+        const minY = clientRect.y - 50;
+
+        if (e.clientX > minX && e.clientX < maxX && e.clientY > minY) {
+            chatbardock.classList.remove('bgaext_hidden');
+        } else {
+            chatbardock.classList.add('bgaext_hidden');
+        }
+    } else {
+        chatbardock = document.querySelector('#chatbardock');
+    }
 };
 
 const initObserver = (page) => {
-	currentObserver = page === 'game' ? initLogObserver(config) : initGameListObserver(config, page);
-	if (!currentObserver) {
-		setTimeout(() => initObserver(page), 500);
-	} else {
-		initChatObserver(config);
-
-		if (!config.isAnimatedTitle()) {
-			initTitleObserver();
-		}
-
-		if (page === 'game' && config.isChatBarAutoHide()) {
-			document.querySelector('body').addEventListener('mousemove', autoHideChat);
-		}
-	}
+    if (page === 'game') {
+        currentObserver = initLogObserver(config);
+    } else if (page === 'gamepanel') {
+        currentObserver = initGamePanelObserver();
+    } else {
+        currentObserver = initGameListObserver(config, page);
+    }
```

Comment:
```suggestion
    if (page === 'game') {
        currentObserver = initLogObserver(config);
    } else if (page === 'gamepanel') {
        currentObserver = initGamePanelObserver();
    } else {
        currentObserver = initGameListObserver(config, page);
    }
```
Should be the only changes besides indentation (auto indentation by vs code I guess, let me know if you want me to revert them)

Reply:
Yep, if you could try to have the same indentation than me (I hope the config file I pushed will help) it would be better for the future :)
Thanks!